e2e/tests/conceptsets/conceptset-edit.spec.ts

```diff
@@ -98,28 +95,14 @@ describe('Editing and Copying Concept Sets', () => {
   /**
    * Test:
    * - Copy Concept Set from one workspace to another workspace.
-   * Note: Use existing workspaces and Concept Set when possible. Otherwise create new workspaces and Concept Sets.
    */
   test('Copy Concept Set to another workspace', async () => {
-    // Find all workspaces.
-    const workspaceCard = new WorkspaceCard(page);
-    const allWorkspaces = await workspaceCard.getWorkspaceMatchAccessLevel(WorkspaceAccessLevel.Owner);
-
-    // Need two workspaces.
-    let workspace1: WorkspaceCard;
-    let workspace2: WorkspaceCard;
-
-    if (allWorkspaces.length > 1) {
-      workspace1 = allWorkspaces[0];
-      workspace2 = allWorkspaces[1];
-    } else {
-      // Don't have two. Create two new workspaces.
-      workspace1 = await findWorkspace(page, true);
-      workspace2 = await findWorkspace(page, true);
-    }
-
-    // Note: workspace1 is the Copy to workspace. workspace2 is the Copy from workspace.
+
```

Comment:
Changes here also for running test consistently in both env. In test env, all existing workspaces were created using Synthetic Dataset 3. In staging env, most of existing workspaces were created on Synthetic Dataset 1. Reusing Synthetic Dataset 1 workspaces are causing problems. So the solution is modify the test code - create new workspaces.